source/_docs/automation/advanced-actions.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "Advanced Actions"
+description: "Automations can have conditionals"
+---
+
+Sometimes to keep things organized in one automation it might be needed to add logic to the action. 
```

Comment:
Why is this a new page? And why is the choose action an advanced action?

As a matter of fact, we don't even document actions for automation, as it refers to the extensive documentation on the script syntax for the actions.

So this page feels a bit added at random. If any place, I think we should adjust the current automation action page; or, figure out the problem with the current script actions page and fix that.

Reply:
It's a new page because I was hesitant if it could be included in the normal actions page(like I mentioned in the PR).
Since it's more than just a basic flow (e.g. something triggers > action), and for beginners who don't have experience with programming/automations it might seem advanced. 

> As a matter of fact, we don't even document actions for automation, as it refers to the extensive documentation on the script syntax for the actions.

And I think this is the reason a lot of users are missing these cool features(myself included). If I'm creating a `automation` I don't expect to lookup documentation stuff related to `scripts`. 

The main thing I was trying to accomplish is to have a clear documentation in regards to `if/else` statements within automations. I couldn't find this myself, maybe because it's named `choose` and not an actual `if/else` statement, so my search query was wrong. To help others in the future, I wanted to help clarify the docs.

Reply:
> And I think this is the reason a lot of users are missing these cool features(myself included). If I'm creating a automation I don't expect to lookup documentation stuff related to scripts.

We should explain that in that case. The new page isn't helping with that imho.

> The main thing I was trying to accomplish is to have a clear documentation in regards to if/else statements within automations. I couldn't find this myself, maybe because it's named choose and not an actual if/else statement, so my search query was wrong. To help others in the future, I wanted to help clarify the docs.

But that is fixing one single case, on hidden away additional page. I think that is not the right thing to do.

Reply:
>We should explain that in that case. The new page isn't helping with that imho.

Agreed, this would add more clutter.
I'll see if I can rework it, and clarify that the `scripts` section can also be used in `automations`

Do you want me to create a new PR when done? Or Should I use this one?

Reply:
Either way it is fine 👍